_summaries/sutskever2014sequence_2.md

```diff
@@ -1,19 +1,79 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{Sutskever2014sequence}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: trowk
+score: 6
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+### What is the core idea?
+
+DNN are models that perform very well on a variety of different tasks. However,
+these models struggle with sequences, problems where the inputs and outputs are
+not fixed. Instead of DNNs, we can use two different LSTMs: one to encode the
+sequence of inputs into a vector representation and another to decode this
+vector into an output sequence.
+
+###  How is it realized (technically)?
+
+Recurrent Neural Networks (RNNs) compute a sequence of outputs from a sequence
+of inputs via the following equations
+
+$$
+h_t =   sigm(W^{hx}x_t + W^{hh}h_{t-1})
+$$
+
+
+$$
+y_t = W^{yh}h_t
+$$
+
+Long Short-Term Memory (LSTM) models are a variant of RNNs that allow the model
+to learn long-term dependencies.The goal of the LSTMs is to approximate
+
+$$
+p(y_1,...,y_{T'}|x_1,...,x_{T}) = \Pi_{t=1}^{T'}p(y_t|v,y_1,...,y_{t-1})
+$$
+
+where the probability distribution is created over all words in the LSTMs
+vocabulary. 
+
+When outputting the translation after reading tin the input, the authors used a
+left-to-right beam search decoder. At each step of the LSTM, the model
+maintained $$B$$ hypotheses on the most likely outputs of the LSTM. The next
+step would then find the $$B$$ most-likely outputs after outputting another word
+onto these $$B$$ hypotheses. By limiting the number of hypotheses to the $$B$$
+most likely, the authors could increase the model's speed with little penalty to
+overall performance.
```

Comment:
The analysis of motivations for beam search and the subsequent design trade-off is great.

Reply:
Thanks!